config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -702,7 +702,7 @@ presubmits:
           - '--node-test-args=--feature-gates=SidecarContainers=true --service-feature-gates=SidecarContainers=true --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
           - --node-tests=true
           - --provider=gce
-          - --test_args=--nodes=1 --timeout=4h --focus="\[Serial\].*\[NodeFeature:SidecarContainers\]|\[NodeFeature:SidecarContainers\].*\[Serial\]" --skip="\[Flaky\]|\[Benchmark\]|\[NodeSpecialFeature:.+\]|\[NodeSpecialFeature\]|\[NodeFeature:Eviction\]"
+          - --test_args=--nodes=1 --timeout=4h --label-filter='Serial && NodeFeature: containsAny SidecarContainers && !Flaky && !Benchmark && NodeSpecialFeature: isEmpty && !(NodeFeature: containsAny Eviction)'
```

Comment:
4 tests with old and new flags:
```console
kubernetes$ _output/bin/ginkgo --dry-run --silence-skips -v --label-filter='Serial && NodeFeature: containsAny SidecarContainers && !Flaky && !Benchmark && NodeSpecialFeature: isEmpty && !(NodeFeature: containsAny Eviction)' ./test/e2e_node/
...
[sig-node] Device Plugin [Feature:DevicePluginProbe] [NodeFeature:DevicePluginProbe] [Serial] DevicePlugin [Serial] [Disruptive] Can schedule a pod with a restartable init container [NodeFeature:SidecarContainers] [sig-node, Feature:DevicePluginProbe, NodeFeature:DevicePluginProbe, Serial, Disruptive, NodeFeature:SidecarContainers]
/nvme/gopath/src/k8s.io/kubernetes/test/e2e_node/device_plugin_test.go:610
• [0.000 seconds]
------------------------------
[sig-node] CPU Manager [Serial] [Feature:CPUManager] With kubeconfig updated with static CPU Manager policy run the CPU Manager tests should not reuse CPUs of restartable init containers [NodeFeature:SidecarContainers] [sig-node, Serial, Feature:CPUManager, NodeFeature:SidecarContainers]
/nvme/gopath/src/k8s.io/kubernetes/test/e2e_node/cpu_manager_test.go:713
• [0.000 seconds]
------------------------------
[sig-node] [NodeFeature:SidecarContainers] [Serial] Containers Lifecycle should restart the containers in right order after the node reboot [sig-node, NodeFeature:SidecarContainers, Serial]
/nvme/gopath/src/k8s.io/kubernetes/test/e2e_node/container_lifecycle_test.go:3128
• [0.000 seconds]
------------------------------
[sig-node] POD Resources [Serial] [Feature:PodResources] [NodeFeature:PodResources] with SRIOV devices in the system with CPU manager Static policy  should return the expected responses [NodeFeature:SidecarContainers] [sig-node, Serial, Feature:PodResources, NodeFeature:PodResources, NodeFeature:SidecarContainers]
/nvme/gopath/src/k8s.io/kubernetes/test/e2e_node/podresources_test.go:901
• [0.000 seconds]
------------------------------
```

Reply:
nice - this is much easier to read and grok.

Reply:
indeed

Reply:
Quoting turned out to be a bit tricky: without quoting the string, YAML treats it as an object because of the ":".

Not a big problem, just something to remember...